less/cards.less

```diff
@@ -16,6 +23,18 @@
   margin-bottom: 15px;
   outline: none;
   cursor:pointer;
+  
+  &:active {
+    -webkit-transition: .2s ease-in-out;
+    transition: 0.2s ease-in-out;
+    outline: none;
+  }
+  
+  &:hover  {
+    background-color: #EFEFEF;
```

Comment:
Didn't like the mixin suggestion?
I think it would improve maintenance and readability of the code. Something like

    .hovered-card() {
      background-color: #EFEFEF;
      transition: 0.1s ease-in-out;
    }

    &:hover {
      .hovered-card;
    }
    
    .highlght .post {
      .hovered-card;
    }

Also, why is the cursor set to pointer hen hovering and highlight class is applied? Either set it only when hovered, or always.

Reply:
I don't know I can do this ;-)